ng add @angular/elements nom ERR! Failed at the document-register-element@1.14.3 postinstall script.

<!--🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅

Oh hi there! 😄

To expedite issue processing please search open and closed issues before submitting a new one.
Existing issues often contain information about workarounds, resolution, or progress updates.

🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅-->


# 🐞 Bug report

### Command (mark with an `x`)
- [ x] add

### Is this a regression?
<!-- Did this behavior use to work in the previous version? -->
<!-- ✍️--> Unknown

### Description
<!-- ✍️-->ng add @angular/elements package install failed

## 🔬 Minimal Reproduction
On Mac/OSX 10.14.6

## 🔥 Exception or Error

Installed packages for tooling via npm.
    Added `document-register-element` as a dependency.
    Added document-register-element.js polyfill to scripts
UPDATE package.json (1376 bytes)
UPDATE angular.json (3790 bytes)

> document-register-element@1.14.3 postinstall /Users/gord/Developer/angular-workspace/projects/pravinkumar-dabade/angular7:8-elements/ng-elements/hello-world2/node_modules/document-register-element
> lightercollective

sh: lightercollective: command not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! document-register-element@1.14.3 postinstall: `lightercollective`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the document-register-element@1.14.3 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/gord/.npm/_logs/2019-11-09T15_01_48_051Z-debug.log
Package install failed, see above.

## 🌍 Your Environment
Node: 12.13.0
Npm: 6.11.3

Angular CLI: 8.3.18
Node: 12.13.0
OS: darwin x64
Angular: 8.2.13
... animations, common, compiler, compiler-cli, core, elements
... forms, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.803.18
@angular-devkit/build-angular     0.803.18
@angular-devkit/build-optimizer   0.803.18
@angular-devkit/build-webpack     0.803.18
@angular-devkit/core              8.3.18
@angular-devkit/schematics        8.3.18
@angular/cli                      8.3.18
@ngtools/webpack                  8.3.18
@schematics/angular               8.3.18
@schematics/update                0.803.18
rxjs                              6.4.0
typescript                        3.5.3
webpack                           4.39.2
</code></pre>

**Anything else relevant?**
npm install @angular/elements completes without error where as ng add @angular/elements throws an error?

--- a/packages/angular/cli/tasks/install-package.ts
+++ b/packages/angular/cli/tasks/install-package.ts
@@ -21,7 +21,6 @@ interface PackageManagerOptions {
   saveDev: string;
   install: string;
   prefix: string;
-  noBinLinks: string;
   noLockfile: string;
 }
 
@@ -39,7 +38,6 @@ export function installPackage(
     packageManagerArgs.install,
     packageName,
     packageManagerArgs.silent,
-    packageManagerArgs.noBinLinks,
   ];
 
   logger.info(colors.green(`Installing packages for tooling via ${packageManager}.`));
@@ -153,7 +151,6 @@ function getPackageManagerArguments(packageManager: PackageManager): PackageMana
         saveDev: '--dev',
         install: 'add',
         prefix: '--modules-folder',
-        noBinLinks: '--no-bin-links',
         noLockfile: '--no-lockfile',
       }
     : {
@@ -161,7 +158,6 @@ function getPackageManagerArguments(packageManager: PackageManager): PackageMana
         saveDev: '--save-dev',
         install: 'install',
         prefix: '--prefix',
-        noBinLinks: '--no-bin-links',
         noLockfile: '--no-package-lock',
       };
 }

